
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genes-and-health.github.io/test-docs/getting-started-as-a-new-user/">
      
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.20">
    
    
      
        <title>Getting started as a new user - Genes & Health TRE Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.e53b48f4.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../stylesheets/extra.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#getting-started-as-a-new-user" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Genes &amp; Health TRE Documentation" class="md-header__button md-logo" aria-label="Genes & Health TRE Documentation" data-md-component="logo">
      
  <img src="../images/branding/genes-and-health-logo-green.svg" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Genes & Health TRE Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Getting started as a new user
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="indigo" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/genes-and-health/test-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Genes &amp; Health TRE Documentation" class="md-nav__button md-logo" aria-label="Genes & Health TRE Documentation" data-md-component="logo">
      
  <img src="../images/branding/genes-and-health-logo-green.svg" alt="logo">

    </a>
    Genes & Health TRE Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/genes-and-health/test-docs" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../what-is-a-tre/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    What is a TRE ?
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Getting started as a new user
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Getting started as a new user
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="steps-to-get-access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Getting access and setting up a profile
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../sandboxes-and-what-each-sandbox-can-access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Sandboxes and what each sandbox can access
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="starting-the-tre/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Starting the TRE
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="choosing-your-virtual-machine/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Choosing your required virtual machine configuration
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Using the TRE
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Using the TRE
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../using-the-tre/understanding-tre-folders-and-buckets/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Understanding TRE folders and buckets
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../using-the-tre/handling-files-and-folders/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Handling files and folders
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../using-the-tre/running-a-gwas/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Running a GWAS
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Getting data in and out of the TRE
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Getting data in and out of the TRE
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../getting-data-in-and-out-of-the-TRE/how-to-get-your-data-in/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    How to get your data in
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../getting-data-in-and-out-of-the-TRE/requesting-data-out-of-the-TRE/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Requesting data out of the TRE
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../latest-tre-data-updates-and-changes/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Latest TRE data updates and changes
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../what-genetic-and-health-data-is-where/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    What genetic and health data is where
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../software%2C-modules%2C-requesting-installs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Software, modules, requesting installs
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../how-to-use-docker/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    How to use docker
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../how-to-write-hash-symbols/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    How to write hash symbols
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../how-to-increase-disk-size/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    How to increase disk size​
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../how-to-update-this-documentation/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    How to update this documentation
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#steps-to-get-access" class="md-nav__link">
    <span class="md-ellipsis">
      Steps to get access
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#setting-up-a-profile" class="md-nav__link">
    <span class="md-ellipsis">
      Setting up a profile
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#starting-the-tre" class="md-nav__link">
    <span class="md-ellipsis">
      Starting the TRE
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Starting the TRE">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#choosing-your-required-machine-configuration" class="md-nav__link">
    <span class="md-ellipsis">
      Choosing your required machine configuration
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#choosing-a-custom-machine-configuration" class="md-nav__link">
    <span class="md-ellipsis">
      Choosing a custom machine configuration
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
        <li class="md-nav__item">
  <a href="#switching-off-the-machine" class="md-nav__link">
    <span class="md-ellipsis">
      Switching off the machine
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#gwas-training-video" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS Training Video
    </span>
  </a>
  
    <nav class="md-nav" aria-label="GWAS Training Video">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#gwas-example" class="md-nav__link">
    <span class="md-ellipsis">
      GWAS Example
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
    <a href="https://github.com/genes-and-health/test-docs/edit/main/docs/getting-started-as-a-new-user.md" title="Edit this page" class="md-content__button md-icon" rel="edit">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4zm10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1z"/></svg>
    </a>
  
  


<h1 id="getting-started-as-a-new-user">Getting started as a new user<a class="headerlink" href="#getting-started-as-a-new-user" title="Permanent link">&para;</a></h1>
<h2 id="steps-to-get-access">Steps to get access<a class="headerlink" href="#steps-to-get-access" title="Permanent link">&para;</a></h2>
<ul>
<li>
<p>All users must read and accept the <a href="https://e4c14f23.tre-documentation.pages.dev/assets/files/TRE_User_Code_of_Conduct_5.0-4fb0174879dd67ccf5dc610fd014d9c2.pdf">Genes &amp; Health Code of Conduct</a>.</p>
</li>
<li>
<p>All users must complete the <a href="https://www.e-lfh.org.uk/programmes/data-security-awareness/">NHS Data Security Awareness Level 1</a> training before they're granted access to Genes and Health data.</p>
</li>
</ul>
<p>Users with an academic email address (.ac.uk) or an NHS email address (@nhs.net) can register for the NHS Data Security Awareness Level 1 training at <a href="https://portal.e-lfh.org.uk/Component/Details/544034">e-learning for Healthcare</a>.  </p>
<div class="admonition note">
<p class="admonition-title">Wellcome Sanger Institute employees</p>
<p>If you're a <a href="https://www.sanger.ac.uk/">Wellcome Sanger Institute</a> employee, please use ODS Code <strong>8J947</strong> and select <strong>Genome Research Limited</strong> from the list of organisations when you register for the course.  </p>
</div>
<p>Non-academic users can register and pay (about £30) for the training course at <a href="https://www.eintegrity.org/healthcare-course/data-security-awareness-from-smc/">eIntegrity Healthcare e-learning</a>. Genes &amp; Health industry consortium users can request a pre-paid link to eIntegrity from us.</p>
<p>After completing the NHS Data Security Awareness Level 1 training course, all users must attest to the <a href="https://e4c14f23.tre-documentation.pages.dev/assets/files/TRE_User_Code_of_Conduct_5.0-4fb0174879dd67ccf5dc610fd014d9c2.pdf">Genes &amp; Health Code of Conduct</a> and upload their certificate to Genes &amp; Health new user registration system. Please contact Genes and Health at genesandhealth@qmul.ac.uk for instructions on how to access the new user registration system.  </p>
<p>These steps are important for our  <a href="https://digital.nhs.uk/data-and-information/looking-after-information/data-security-and-information-governance/data-security-and-protection-toolkit">NHS Digital Data Security and Protection Toolkit</a>, and <a href="https://www.iso.org/standard/27001">ISO 27001</a> accreditation.  </p>
<div class="admonition warning">
<p class="admonition-title">Caution</p>
<p>Users who have not completed the Data Security Awareness training or attested to the Code of Conduct are subject to having their account deactivated.</p>
</div>
<h2 id="setting-up-a-profile">Setting up a profile<a class="headerlink" href="#setting-up-a-profile" title="Permanent link">&para;</a></h2>
<p>When first set up, users will get an email from Google (to their normal email address). This will have a username@genesandhealth.qmul.ac.uk username. You need to Add this as a Profile in Google Chrome. Please do not try to use any other browser than Google Chrome!  </p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>This username is not an email address, and we will not send email to it. It cannot receive email; it is just a username for the TRE.  Once you have done this, go to Manage Your Account and Security and turn 2 Step Verification on. You will need to enter a phone number and get a code by SMS/text message. You cannot access the TRE without this enabled.</p>
</div>
<h2 id="starting-the-tre">Starting the TRE<a class="headerlink" href="#starting-the-tre" title="Permanent link">&para;</a></h2>
<p>Having selected your username@genesandhealth.qmul.ac.uk account as your Chrome Profile, enter the G&amp;H TRE’s URL in your Chrome browser. Your URL will start with <a href="https://new-production.genesandhealth.qmul.ac.uk/">https://new-production.genesandhealth.qmul.ac.uk/</a>. The full URL is dependent on your sandbox and can be obtained from the table below.<br />
This will take you to the TRE login page. You will be asked to enter your username and password. The username is the one you have been given by the Genes and Health team, and the password is the one you have set up when you first logged in.<br />
If you want to go a specific sandbox, you can use the following URL table:</p>
<table>
<thead>
<tr>
<th style="text-align: left;">Sandbox</th>
<th style="text-align: left;">URL</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;">Sandbox 1 - QMUL +WSI Core Team Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 2 - External Academic Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-2/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 3 - GSK Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-3/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 4 - BMS Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-4/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 5 - MSD Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-5/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 6 - Takeda Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-6/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 7 - Pfizer Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-7/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 8 - S00050_FFAIR-PRS Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-8/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 9 - Maze Therapeutics Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-9/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 10 - Novo Nordisk Desktop</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-10/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox  11 - University of Exter</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-11/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 13 - AstraZeneca</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-13/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 14 - External Academic, Consortium access</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-14/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 15 - 5 Prime Sciences</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-15/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 16 - Sandbox 16</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-16/vm</a></td>
</tr>
<tr>
<td style="text-align: left;">Sandbox 17 - Academic, NHS Digital access</td>
<td style="text-align: left;"><a href="https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-1/vm">https://new-production.genesandhealth.qmul.ac.uk/qmul-production-sandbox-17/vm</a></td>
</tr>
</tbody>
</table>
<h3 id="choosing-your-required-machine-configuration">Choosing your required machine configuration<a class="headerlink" href="#choosing-your-required-machine-configuration" title="Permanent link">&para;</a></h3>
<p>Once logged in, you will be taken to the <strong>Virtual Machines</strong> page where you can start a new machine or connect to an existing one.<br />
<img alt="Virtual Machine options" src="images/sandboxes-and-folder-structures/virtual-machine-options.png" />
You will be offered a variety of virtual machine types.  </p>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>Choose the Basic machine unless a script requires high memory or increased CPU capacity, and is ready to go. Test your script on the Basic machine first, as other options are more expensive.  </p>
</div>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>Some machine types have many CPUs, which are suitable for multithreaded applications like plink or regenie that can fully utilise them. These applications may sometimes run input/output operations too quickly for standard Google bucket storage, requiring you to copy key files to faster local storage (see below).</p>
</div>
<h3 id="choosing-a-custom-machine-configuration">Choosing a custom machine configuration<a class="headerlink" href="#choosing-a-custom-machine-configuration" title="Permanent link">&para;</a></h3>
<p>It is possible that sometimes you may require a Virtual Machine that offers resources different to the standard flavors i.e. higher-memory, higher-cpu or memory-optimised instances. You can choose a custom VM configuration from the <strong>Virtual Machines</strong> page by expanding the <strong>Use custom Virtual Machine profile</strong> section at the bottom.<br />
<img alt="Custom Virtual Machine profile" src="images/sandboxes-and-folder-structures/custom-virtual-machine-profile.png" /></p>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>Choose the Use custom Virtual Machine profile only when absolutely necessary, use Basic machine unless a script requires high memory or increased CPU capacity. Excess use of higher-spec machines can significantly impact running costs. Test your script on the Basic machine first, as other options are more expensive.</p>
</div>
<h2 id="switching-off-the-machine">Switching off the machine<a class="headerlink" href="#switching-off-the-machine" title="Permanent link">&para;</a></h2>
<p>Your virtual machine will keep running for 24 hours if you are doing nothing. You can disconnect from Chrome, turn your laptop off, and then you will be straight back into the exact same machine when you connect again (within 24h). If you have Linux jobs running, the machine will keep going while these run (however long) plus another 48h.  </p>
<div class="admonition warning">
<p class="admonition-title">Warning</p>
<p>Keeping machines running in the background costs money. So please actively shutdown (if you do not need the extra 48h) using the "Off" button on the bottom of the side menu.</p>
</div>
<h2 id="gwas-training-video">GWAS Training Video<a class="headerlink" href="#gwas-training-video" title="Permanent link">&para;</a></h2>
<p>Genes &amp; Health have provided a number of training videos to help researchers get started with the Trusted Research Environment (TRE). This is a 45-min tutorial in which GWAS using regenie is demonstrated (300Mb with mp4 format, updated 5 oct 2022). This video is a recording of a live demo session, and the quality may not be perfect. It is recommended to watch the video in full screen mode. Please feel free to download the video and watch it at your convenience.<br />
<a href="https://dl.dropboxusercontent.com/s/k6h5botp53xk3l6/TRE%20Training-20220928_160450-Meeting%20Recording.mp4?e=1&amp;dl=0">https://dl.dropboxusercontent.com/s/k6h5botp53xk3l6/TRE%20Training-20220928_160450-Meeting%20Recording.mp4?e=1\&amp;dl=0</a></p>
<h3 id="gwas-example">GWAS Example<a class="headerlink" href="#gwas-example" title="Permanent link">&para;</a></h3>
<p>You can find an example regenie script for binary trait GWAS below. Please feel free to adapt and reuse this script.</p>
<div class="admonition warning">
<p class="admonition-title">Caution</p>
<p>Please let your script run first on the cheap 2-core machine. Regenie has excellent multithreading capabilities and will perform best on the 64-core high-performance machine, but this option is expensive. The 2-core machine is sufficiently fast on a large multicore machine, allowing you to run it directly from the command line using a single all-chromosome pgen input file, without the need for HPC or splitting by chromosome. Regenie will utilise all available threads minus 1 by default, meaning the 64-core machine will run 63 times faster for the compute steps compared to the 2-core machine. It is fast enough on a large multicore machine to run several phenotypes. If you intend to run 1000 phenotypes, you will need to use Google HPC / WDL.</p>
</div>
<div class="language-bash highlight"><pre><span></span><code><span id="__span-0-1"><a id="__codelineno-0-1" name="__codelineno-0-1" href="#__codelineno-0-1"></a><span class="c1">## RUN THIS FROM THE FOLDER WITH THE </span>
</span><span id="__span-0-2"><a id="__codelineno-0-2" name="__codelineno-0-2" href="#__codelineno-0-2"></a><span class="c1">## COVAR/PHENO FILES AND WHERE YOU WANT THE OUTPUT</span>
</span><span id="__span-0-3"><a id="__codelineno-0-3" name="__codelineno-0-3" href="#__codelineno-0-3"></a>
</span><span id="__span-0-4"><a id="__codelineno-0-4" name="__codelineno-0-4" href="#__codelineno-0-4"></a><span class="c1">## this is for BINARY TRAITS</span>
</span><span id="__span-0-5"><a id="__codelineno-0-5" name="__codelineno-0-5" href="#__codelineno-0-5"></a><span class="c1">## remove the  --bt and firth etc options for quant traits, see the online regenie manual https://rgcgithub.github.io/regenie/options/</span>
</span><span id="__span-0-6"><a id="__codelineno-0-6" name="__codelineno-0-6" href="#__codelineno-0-6"></a>
</span><span id="__span-0-7"><a id="__codelineno-0-7" name="__codelineno-0-7" href="#__codelineno-0-7"></a><span class="c1">## first copy big files to local storage ivm SSD for better i/o than with google storage buckets</span>
</span><span id="__span-0-8"><a id="__codelineno-0-8" name="__codelineno-0-8" href="#__codelineno-0-8"></a><span class="c1">## regenie will sometimes crash if reading direct from library-red due to slow i/o</span>
</span><span id="__span-0-9"><a id="__codelineno-0-9" name="__codelineno-0-9" href="#__codelineno-0-9"></a><span class="c1">## cp /genesandhealth/library-red/genesandhealth/GSAv3EAMD/Jul2021_44k_TOPMED-r2_Imputation_b38/topmed-r2_merged_version03/chrALLincX.dose.merged_INFO0.3_MAF0.00001_F_MISSING0.2.8bit.sorted.p* /home/ivm/Documents/</span>
</span><span id="__span-0-10"><a id="__codelineno-0-10" name="__codelineno-0-10" href="#__codelineno-0-10"></a>
</span><span id="__span-0-11"><a id="__codelineno-0-11" name="__codelineno-0-11" href="#__codelineno-0-11"></a><span class="c1">## CHANGE THE DATE EACH RUN ## </span>
</span><span id="__span-0-12"><a id="__codelineno-0-12" name="__codelineno-0-12" href="#__codelineno-0-12"></a><span class="nv">daterun</span><span class="o">=</span>2022_07_20
</span><span id="__span-0-13"><a id="__codelineno-0-13" name="__codelineno-0-13" href="#__codelineno-0-13"></a>
</span><span id="__span-0-14"><a id="__codelineno-0-14" name="__codelineno-0-14" href="#__codelineno-0-14"></a><span class="nv">covarfile</span><span class="o">=</span><span class="s2">&quot;/genesandhealth/library-red/genesandhealth/GSAv3EAMD/Jul2021_44k_TOPMED-r2_Imputation_b38/GNH.44190.noEthnicOutliers.covariates.20PCs.tab&quot;</span>
</span><span id="__span-0-15"><a id="__codelineno-0-15" name="__codelineno-0-15" href="#__codelineno-0-15"></a><span class="c1">## updated to covarfile with correct FID IID structure 15 July 2022</span>
</span><span id="__span-0-16"><a id="__codelineno-0-16" name="__codelineno-0-16" href="#__codelineno-0-16"></a>
</span><span id="__span-0-17"><a id="__codelineno-0-17" name="__codelineno-0-17" href="#__codelineno-0-17"></a><span class="nv">phenofile</span><span class="o">=</span><span class="s2">&quot;/genesandhealth/library-red/genesandhealth/phenotypes_curated/version005_2022_06/1stoccurrence_3digitICD10_1SNOMEDto1ICD10/2022_06_version005_3digitICD10_1to1_42029withbothICD10andGSAJul2021.txt&quot;</span>
</span><span id="__span-0-18"><a id="__codelineno-0-18" name="__codelineno-0-18" href="#__codelineno-0-18"></a>
</span><span id="__span-0-19"><a id="__codelineno-0-19" name="__codelineno-0-19" href="#__codelineno-0-19"></a><span class="c1">## these are the custom phenotypes: &quot;/genesandhealth/library-red//genesandhealth/phenotypes_curated/version005_2022_06/custom/2022-06-15_big_regenie_phenoFile.txt&quot;</span>
</span><span id="__span-0-20"><a id="__codelineno-0-20" name="__codelineno-0-20" href="#__codelineno-0-20"></a>
</span><span id="__span-0-21"><a id="__codelineno-0-21" name="__codelineno-0-21" href="#__codelineno-0-21"></a><span class="nv">grmfile</span><span class="o">=</span><span class="s2">&quot;/genesandhealth/library-red/genesandhealth/GSAv3EAMD/Jul2021_44k_TOPMED-r2_Imputation_b38/bfile_forSAIGEgrm_44396_chipgenotypes_indep-pairwise_500_50_0.2_LDpruned_NotChrY&quot;</span>
</span><span id="__span-0-22"><a id="__codelineno-0-22" name="__codelineno-0-22" href="#__codelineno-0-22"></a>
</span><span id="__span-0-23"><a id="__codelineno-0-23" name="__codelineno-0-23" href="#__codelineno-0-23"></a><span class="c1">## regenie will run all phenotypes in the phenoFIle if you dont specify --phenoCol or --phenoColList</span>
</span><span id="__span-0-24"><a id="__codelineno-0-24" name="__codelineno-0-24" href="#__codelineno-0-24"></a>
</span><span id="__span-0-25"><a id="__codelineno-0-25" name="__codelineno-0-25" href="#__codelineno-0-25"></a><span class="c1">## beware the .log file does not have phenotype name in its filename (because it is designed for running many phenotypes at once), can get overwritten if run multiple times using same $daterun</span>
</span><span id="__span-0-26"><a id="__codelineno-0-26" name="__codelineno-0-26" href="#__codelineno-0-26"></a>
</span><span id="__span-0-27"><a id="__codelineno-0-27" name="__codelineno-0-27" href="#__codelineno-0-27"></a><span class="c1">## if want to run male-only or female-only analysis, best to use --keep with a list of individuals in Step1 Step2 rather than --sex-specific. Because sex-specific uses the sex in the bed or pgen files not from the covariate file. And sex is NA in the pgen file we have made</span>
</span><span id="__span-0-28"><a id="__codelineno-0-28" name="__codelineno-0-28" href="#__codelineno-0-28"></a>
</span><span id="__span-0-29"><a id="__codelineno-0-29" name="__codelineno-0-29" href="#__codelineno-0-29"></a>regenie<span class="w">   </span><span class="se">\</span>
</span><span id="__span-0-30"><a id="__codelineno-0-30" name="__codelineno-0-30" href="#__codelineno-0-30"></a><span class="w">  </span>--step<span class="w"> </span><span class="m">1</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-31"><a id="__codelineno-0-31" name="__codelineno-0-31" href="#__codelineno-0-31"></a><span class="w">  </span>--bed<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$grmfile</span><span class="s2">&quot;</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-32"><a id="__codelineno-0-32" name="__codelineno-0-32" href="#__codelineno-0-32"></a><span class="w">  </span>--covarFile<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$covarfile</span><span class="s2">&quot;</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-33"><a id="__codelineno-0-33" name="__codelineno-0-33" href="#__codelineno-0-33"></a><span class="w">  </span>--phenoFile<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$phenofile</span><span class="s2">&quot;</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-34"><a id="__codelineno-0-34" name="__codelineno-0-34" href="#__codelineno-0-34"></a><span class="w">  </span>--phenoExcludeList<span class="w"> </span>PseudoNHS_2022_02_08<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-35"><a id="__codelineno-0-35" name="__codelineno-0-35" href="#__codelineno-0-35"></a><span class="w">  </span>--phenoCol<span class="w"> </span>ICD10__E10,ICD10__E11<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-36"><a id="__codelineno-0-36" name="__codelineno-0-36" href="#__codelineno-0-36"></a><span class="w">  </span>--minCaseCount<span class="w"> </span><span class="m">100</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-37"><a id="__codelineno-0-37" name="__codelineno-0-37" href="#__codelineno-0-37"></a><span class="w">  </span>--bsize<span class="w"> </span><span class="m">1000</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-38"><a id="__codelineno-0-38" name="__codelineno-0-38" href="#__codelineno-0-38"></a><span class="w">  </span>--bt<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-39"><a id="__codelineno-0-39" name="__codelineno-0-39" href="#__codelineno-0-39"></a><span class="w">  </span>--lowmem<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-40"><a id="__codelineno-0-40" name="__codelineno-0-40" href="#__codelineno-0-40"></a><span class="w">  </span>--lowmem-prefix<span class="w"> </span>tmp_rg<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-41"><a id="__codelineno-0-41" name="__codelineno-0-41" href="#__codelineno-0-41"></a><span class="w">  </span>--gz<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-42"><a id="__codelineno-0-42" name="__codelineno-0-42" href="#__codelineno-0-42"></a><span class="w">  </span>--verbose<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-43"><a id="__codelineno-0-43" name="__codelineno-0-43" href="#__codelineno-0-43"></a><span class="w">  </span>--out<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$daterun</span><span class="s2">&quot;</span>_fit_binary_out
</span><span id="__span-0-44"><a id="__codelineno-0-44" name="__codelineno-0-44" href="#__codelineno-0-44"></a>
</span><span id="__span-0-45"><a id="__codelineno-0-45" name="__codelineno-0-45" href="#__codelineno-0-45"></a>regenie<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-46"><a id="__codelineno-0-46" name="__codelineno-0-46" href="#__codelineno-0-46"></a><span class="w">  </span>--step<span class="w"> </span><span class="m">2</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-47"><a id="__codelineno-0-47" name="__codelineno-0-47" href="#__codelineno-0-47"></a><span class="w">  </span>--pgen<span class="w"> </span>/home/ivm/Documents/chrALLincX.dose.merged_INFO0.3_MAF0.00001_F_MISSING0.2.8bit.sorted<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-48"><a id="__codelineno-0-48" name="__codelineno-0-48" href="#__codelineno-0-48"></a><span class="w">  </span>--covarFile<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$covarfile</span><span class="s2">&quot;</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-49"><a id="__codelineno-0-49" name="__codelineno-0-49" href="#__codelineno-0-49"></a><span class="w">  </span>--phenoFile<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$phenofile</span><span class="s2">&quot;</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-50"><a id="__codelineno-0-50" name="__codelineno-0-50" href="#__codelineno-0-50"></a><span class="w">  </span>--bsize<span class="w"> </span><span class="m">1000</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-51"><a id="__codelineno-0-51" name="__codelineno-0-51" href="#__codelineno-0-51"></a><span class="w">  </span>--minMAC<span class="w"> </span><span class="m">20</span><span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-52"><a id="__codelineno-0-52" name="__codelineno-0-52" href="#__codelineno-0-52"></a><span class="w">  </span>--minINFO<span class="w"> </span><span class="m">0</span>.6<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-53"><a id="__codelineno-0-53" name="__codelineno-0-53" href="#__codelineno-0-53"></a><span class="w">  </span>--bt<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-54"><a id="__codelineno-0-54" name="__codelineno-0-54" href="#__codelineno-0-54"></a><span class="w">  </span>--firth<span class="w"> </span>--approx<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-55"><a id="__codelineno-0-55" name="__codelineno-0-55" href="#__codelineno-0-55"></a><span class="w">  </span>--pThresh<span class="w"> </span><span class="m">0</span>.01<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-56"><a id="__codelineno-0-56" name="__codelineno-0-56" href="#__codelineno-0-56"></a><span class="w">  </span>--pred<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$daterun</span><span class="s2">&quot;</span>_fit_binary_out_pred.list<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-57"><a id="__codelineno-0-57" name="__codelineno-0-57" href="#__codelineno-0-57"></a><span class="w">  </span>--gz<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-58"><a id="__codelineno-0-58" name="__codelineno-0-58" href="#__codelineno-0-58"></a><span class="w">  </span>--verbose<span class="w"> </span><span class="se">\</span>
</span><span id="__span-0-59"><a id="__codelineno-0-59" name="__codelineno-0-59" href="#__codelineno-0-59"></a><span class="w">  </span>--out<span class="w"> </span><span class="s2">&quot;</span><span class="nv">$daterun</span><span class="s2">&quot;</span>_GNH_binary_GWAS_firth
</span><span id="__span-0-60"><a id="__codelineno-0-60" name="__codelineno-0-60" href="#__codelineno-0-60"></a>
</span><span id="__span-0-61"><a id="__codelineno-0-61" name="__codelineno-0-61" href="#__codelineno-0-61"></a><span class="c1">## END ##</span>
</span></code></pre></div>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["content.code.copy", "content.code.select", "content.code.annotate", "content.action.edit", "navigation.footer", "navigation.path", "navigation.expand", "navigation.instant", "navigation.tracking"], "search": "../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.f55a23d4.min.js"></script>
      
    
  </body>
</html>